### PR TITLE
Klaytn v1.11.0

### PR DESCRIPTION
**Pull request by ayo-klaytn**
## Proposed changes

Klaytn-v1.11.0 added

## Types of changes

Please put an x in the boxes related to your change.

- [x] Minor Issues and Typos
- [ ] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md)
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn-docs)
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large content contribution, kick off the discussion by explaining why you would suggest the content contribution, etc...

### CODE REVIEW

**Review by aidan-kwon — APPROVED** [review]
Great! Thanks!

File: docs/toolkit/klaytn-contracts-wizard.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
-# Klaytn Contracts Wizard
```
Comment (Scott-Klatn) [inline docs/toolkit/klaytn-contracts-wizard.md:1]: Shouldn't this line be kept?

File: docs/toolkit/klaytn-contracts-wizard.md [hunk 2]
```diff
@@ -1,5 +1,3 @@
...
-
 # Introduction
```
Comment (Scott-Klatn) [inline docs/toolkit/klaytn-contracts-wizard.md:1]: "# Introduction" to "## Introduction"

File: docs/toolkit/klaytn-contracts-wizard.md (hunk 1 above)
Comment (ayo-klaytn) [reply docs/toolkit/klaytn-contracts-wizard.md:1]: Yes! But in this case, we can omit it the title in Summary.md will become the heading

[the remark this answers is not in the document]
Comment (ayo-klaytn) [reply docs/toolkit/klaytn-contracts-wizard.md:1]: Yes! But in this case, we can omit it the title in Summary.md will become the heading

**Review by Scott-Klatn — APPROVED** [review]

File: docs/installation-guide/download/v1.11.0.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
+# Package Downloads <a id="package-downloads"></a>
+
+## Linux <a id="linux"></a>
+
+Cypress Packages
```
Comment (yoomee1313) [inline docs/installation-guide/download/v1.11.0.md:5]: Hi @ayo-klaytn, RPM package is not listed in here. If it's not intended one, can I add the RPM package?

Comment (ayo-klaytn) [reply docs/installation-guide/download/v1.11.0.md:5]: Hi @yoomee1313, got the packages from binary section of the the release note [here](https://github.com/klaytn/klaytn/releases/tag/v1.11.0). If it exists you can add the RPM package. 

Thanks for the observation 🙏

Comment (yoomee1313) [reply docs/installation-guide/download/v1.11.0.md:5]: I'll add it now :) Actually @kjeom found it. Thanks jack 👍

**Pull request merged**